tests/libs/util/socket_helper.cpp

```diff
@@ -806,3 +806,28 @@ _stream_server_socket::query_redirect_context(_Inout_ void* buffer, uint32_t buf
         nullptr,
         nullptr);
 }
+
+_wsa_helper::_wsa_helper()
```

Comment:
these should just be defined when we declare the class. there's negative value in pulling these out